django/test/runner.py

```diff
@@ -42,7 +42,7 @@
 
 class DebugSQLTextTestResult(unittest.TextTestResult):
     def __init__(self, stream, descriptions, verbosity):
-        self.logger = logging.getLogger('django.db.backends')
+        self.logger = logging.getLogger("django.db.backends")
```

Comment:
Keep the style the same here and below

Reply:
So it looks as though you've run `black` over this as there are changes other than quotes.
We're not currently ready to use `black` as it is still in pre-release.

Reply:
Yes, I did. Will revert all stylistic changes